README.rst

```diff
@@ -0,0 +1,342 @@
+| ``/omp/occa``              | OpenMP OCCA kernels                               |
++----------------------------+---------------------------------------------------+
+| ``/ocl/occa``              | OpenCL OCCA kernels                               |
++----------------------------+---------------------------------------------------+
+| ``/gpu/cuda/ref``          | Reference pure CUDA kernels                       |
++----------------------------+---------------------------------------------------+
+| ``/gpu/cuda/reg``          | Pure CUDA kernels using one thread per element    |
++----------------------------+---------------------------------------------------+
+| ``/gpu/cuda/shared``       | Optimized pure CUDA kernels using shared memory   |
++----------------------------+---------------------------------------------------+
+| ``/gpu/cuda/gen``          | Optimized pure CUDA kernels using code generation |
++----------------------------+---------------------------------------------------+
+| ``/gpu/magma``             | CUDA MAGMA kernels                                |
++----------------------------+---------------------------------------------------+
+
+The ``/cpu/self/*/serial`` backends process one element at a time and are intended for meshes
+with a smaller number of high order elements. The ``/cpu/self/*/blocked`` backends process
+blocked batches of eight interlaced elements and are intended for meshes with higher numbers
+of elements.
+
+The ``/cpu/self/ref/*`` backends are written in pure C and provide basic functionality.
+
+The ``/cpu/self/opt/*`` backends are written in pure C and use partial e-vectors to improve performance.
+
+The ``/cpu/self/avx/*`` backends rely upon AVX instructions to provide vectorized CPU performance.
+
+The ``/cpu/self/xsmm/*`` backends rely upon the `LIBXSMM <http://github.com/hfp/libxsmm>`_ package
+to provide vectorized CPU performance. If linking MKL and LIBXSMM is desired but
+the Makefile is not detecting ``MKLROOT``, linking libCEED against MKL can be
+forced by setting the environment variable ``MKL=1``.
+
+The ``/cpu/self/memcheck/*`` backends rely upon the `Valgrind <http://valgrind.org/>`_ Memcheck tool
+to help verify that user QFunctions have no undefined values. To use, run your code with
+Valgrind and the Memcheck backends, e.g. ``valgrind ./build/ex1 -ceed /cpu/self/ref/memcheck``. A
+'development' or 'debugging' version of Valgrind with headers is required to use this backend.
+This backend can be run in serial or blocked mode and defaults to running in the serial mode
+if ``/cpu/self/memcheck`` is selected at runtime.
+
+The ``/*/occa`` backends rely upon the `OCCA <http://github.com/libocca/occa>`_ package to provide
+cross platform performance.
+
+The ``/gpu/cuda/*`` backends provide GPU performance strictly using CUDA.
+
+The ``/gpu/magma`` backend relies upon the `MAGMA <https://bitbucket.org/icl/magma>`_ package.
+
+## Examples
```

Comment:
```suggestion
Examples
----------------------------------------
```

Reply:
The fact that this remains suggests that you didn't convert it with pandoc -- oh my!

Reply:
No, I didn't. When your suggestion came in on Slack for the other conversion I did (rst -> tex) I was already done ^_^'